src/extra/AutoUI/utils.ts

```diff
@@ -71,3 +77,41 @@ export const autoUIGetDisabledReason = <T extends AutoUIBaseResource<T>>(
 		});
 	}
 };
+
+export const formatSorters: Dictionary<TableSortFunction<any>> = {};
```

Comment:
This sounds wrong. That was an angular 1 service stile. What is the use case ?

Reply:
@JSReds I do not know, this was pre-existing code. I have just moved some things around, which includes bringing this to this file